src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({entries, onToggleLike}) => {
+
+    const ChatComponents = entries.map((message) => {
+        return (
+        <div>
+            <ChatEntry 
+            id={message.id} 
+            sender={message.sender} 
+            body={message.body} 
+            timeStamp={message.timeStamp} 
+            liked={message.liked} 
+            onToggleLike={onToggleLike}/>
+        </div>)
+});
+    return (
+        <div>
+            {ChatComponents}
+        </div>
```

Comment:
Same note - I'd recommend dropping the `<div>` since you are only returning one element